src/app.ts

```diff
@@ -911,6 +911,12 @@ router.get(
   safeRequestHandler(businessesAPI.retrieveBusinessIdentification)
 );
 
+router.get(
+  "businesses/:business_id/accounts/:id",
```

Comment:
Personal preference, not mandatory, yet I think it also aligns with other endpoints

```suggestion
  "businesses/:business_id/accounts/:account_id",
```

Reply:
If applied, the tests will need a small adjustment too 😉